docs/b2b-edition/specs/storefront/storefront/order.yaml

```diff
@@ -351,27 +357,67 @@ paths:
                 isSaveOrderComment:
                   type: string
                   description: If save order comment
-                extraFields:
```

Comment:
`extraFields` should be added back in, as this is what the B2B team is broadcasting as the new standard. You can see this was recently merged in #540